utilities/backupable/backupable_db.cc

```diff
@@ -207,8 +214,8 @@ class BackupEngineImpl : public BackupEngine {
     uint64_t size_;
     std::string const meta_filename_;
     // files with relative paths (without "/" prefix!!)
-    std::vector<std::string> files_;
-    std::unordered_map<std::string, FileInfo>* file_infos_;
+    std::vector<shared_ptr<FileInfo> > files_;
```

Comment:
Files may be shared among backups. Each backup is represented by a BackupMeta. file_infos_ contains files mentioned in all meta files. Need a way to store which backup contains which files.